lib/appium/AppiumRunner.js

```diff
@@ -102,6 +102,24 @@ function getConfigPath(appPath) {
     return path.join(appPath, 'config.xml');
 }
 
+function installAppiumServer() {
+    /*
+     * We install appium in this counter-intuitive way at runtime in order to
+     * not trigger massive dependency downloads via `npm install` for autotest
+     * execution. Installing appium only happens in the local-running case, and
+     * thus is not needed in our CI system. By using this method, we save the CI
+     * a lot of time.
+     * ALTERNATIVE: what if paramedic dependencies were installed via `npm install
+     * --production`, and appium was stuffed into `devDependencies`. That should
+     * also prevent this problem? 
```

Comment:
@purplecabbage Can you elaborate how either of the current approach or putting appium into devDependencies sacrifices the individual plugin developer? I don't follow.